.pre-commit-config.yaml

```
@@ -0,0 +1,30 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+exclude: |
+  (?x)^(
+    {{ cookiecutter.import_name }}/_version.py|
```

```suggestion
    lcls_naming_tool/_version.py|
```

Parts of the cookiecutter were left over